docs/index.rst

```diff
@@ -8,12 +8,155 @@ Welcome to Raiden Services's documentation!
+
+The Raiden services require Python 3.7. To install the latest version of
+the services follow these instructions:
+
+.. code:: bash
+
+    git clone git@github.com:raiden-network/raiden-services.git
+    cd raiden-services
+    virtualenv -p python3.7 venv
+    source venv/bin/activate
+    pip install -e .
+
+This installs the executables ``pathfinding-service``, ``monitoring-service``
+and ``request-collector`` into the virtual environment.
+
+Running the services
+--------------------
+
+The services share a number of options. These are:
+
+``--keystore-file``
+    Defines the path to the keystore file. (*Required*)
+
+``--log-level``
+    Defines the minimal level of log messages to be shown.
+
+    Defaults to ``INFO``.
+
+``--eth-rpc``
+    Defines the URI of the Ethereum node to be used.
+
+    Defaults to ``http://localhost:8545``.
+
+``--registry-address``
+    Defines the address of the token network registry to be used.
+
+    Defaults to the contract address that Raiden client uses for the given
+    chain.
+
+``--user-deposit-contract-address``
+    Defines the address of the user deposit contract to be used.
+
+    Defaults to the contract address that Raiden client uses for the given
+    chain.
+
+``--start-block``
+    Defines the block number at which the service starts syncing.
+
+    Defaults to the block number in which the earliest contract was deployed,
+    but only if only predeployed contracts are used. Otherwise is set to 0.
+
+``--confirmations``
+    Defines the number of blocks after which blocks are assumed confirmed.
+
+    Defaults to 8 blocks.
+
+Pathfinding Service
+^^^^^^^^^^^^^^^^^^^
+
+The Pathfinding service has some additional parameters which can be set.
+
+``--state-db``
+    Defines the location where the state is stored.
+
+    Defaults to ``~/.config/raiden-pathfinding-service/state.db`` on Unix or
+    ``~/Library/Application Support/raiden-pathfinding-service/state.db`` on
+    macOS.
+
+``--host``
+    Defines the URI the REST API of the Pathfinding service will be available
+    at.
+
+    Defaults to ``localhost``.
+
+``--service-fee``
+    Defines the minimum service fee a client has to pay when requesting routes.
+
+    Defaults to zero.
+
+
+Monitoring Service
+^^^^^^^^^^^^^^^^^^
+
+The Monitoring service consists of two components that have to be started
+indenpendently for now. The ``monitoring-service`` program is responsible for
+all interactions with the blockchain and **must** be started first, as it
+writes some essential information to the state database.
+Afterwards the ``request-collector`` can be started. It listens to the public
+Matrix room for monitporing request from Raiden nodes and validates and saves
+them to the monitoring services database.
+
+.. note::
+    The reason of this separation is security. Even when the broadcast room and
+    therefore the request collector are attacked, the monitoring service
+    continues working and can safely monitor the users channels.
```

Comment:
`working` -> `to work`

And do you to mean `users' channels` or `user's channels`?

Reply:
Thanks for all these findings. Have to proof read better in the future.

As for this. I mean the channel that one user is part of, so that should be `a user's channel` right!?

Reply:
No worries. I think it's a good way to do it and it keeps the reviewer's eyes sharp when there are some things to find ;)

But yes, it should be `user's channels` if it's the channels that one user is part of